ecp_integration/integration.py

```diff
@@ -148,7 +148,16 @@ def get_doctors_ecp_free_dates_by_research(research_pk, date_start, date_end, ho
             )
         schedule_data = req_result['data']
         if len(schedule_data) > 0:
-            doctors_has_free_date[d.rmis_location] = {"fio": f"{d.family} {d.name} {d.patronymic}", "pk": d.id, "dates": []}
+            message = ""
+            if d.age_limit > -1:
+                years = d.age_limit // 12
+                months = d.age_limit % 12
+                if months > 0:
+                    message = f"не старше {years}г {months} мес"
+                else:
+                    message = f"не старше {years}г"
+            district = d.district_title if d.district_title else ""
+            doctors_has_free_date[d.rmis_location] = {"fio": f"{district} {d.family} {d.name[:1]}. {d.patronymic[:1]} {message}", "pk": d.id, "dates": [], "message_age_limit": message}
```

Comment:
Странно, что сокращения фио не одинаковые. Плюс есть лишний пробел, если нет ограничения по возрасту
```suggestion
            doctors_has_free_date[d.rmis_location] = {"fio": f"{district} {d.family} {d.name[:1]}. {d.patronymic[:1]}. {message}".strip(), "pk": d.id, "dates": [], "message_age_limit": message}
```